filters/all.js

```diff
@@ -2,7 +2,15 @@ const filter = module.exports;
 const _ = require('lodash');
 
 function defineType(prop, propName) {
-    if (prop.type() === 'object') {
+    if (prop.additionalProperties()) {
```

Comment:
Note: Implicit check, if `prop.additionalProperties()` is either `false` or `undefined`/`null`

Reply:
fixed by me